internal/app/s3manager/bucket_view_test.go

```diff
@@ -11,11 +11,12 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/cloudlena/s3manager/internal/app/s3manager"
-	"github.com/cloudlena/s3manager/internal/app/s3manager/mocks"
+	"github.com/gorilla/mux"
 	"github.com/matryer/is"
-	"github.com/matryer/way"
 	"github.com/minio/minio-go/v7"
+
```

Comment:
This blank line was added manually and not by `goimports`. It should therefore be omitted.

Reply:
Fixed. Used gci instead of goimports